package.json

```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/KeitIG/museeks"
   },
-  "author": "Pierre de la Martinière",
+  "author": "Pierre de la Martinière <pierre.de.la.martiniere@gmail.com>",
```

Comment:
Can be split into multiple fields: https://docs.npmjs.com/files/package.json#people-fields-author-contributors